www/src/components/email-capture-form.js

```diff
@@ -0,0 +1,145 @@
+import React from "react"
+import { rhythm } from "../utils/typography"
+import presets from "../utils/presets"
+import jsonp from "jsonp"
+
+// Mailchimp endpoint
+// From: https://us17.admin.mailchimp.com/lists/integration/embeddedcode?id=XXXXXX
+// Where `XXXXXX` is the MC list ID
+// Note: we change `/post` to `/post-json`
+const MAILCHIMP_URL = `https://gatsbyjs.us17.list-manage.com/subscribe/post-json?u=1dc33f19eb115f7ebe4afe5ee&amp;id=f366064ba7`
```

Comment:
should i make these env vars?

Reply:
ideally this should be a prop passed into the component named mailChimpUrl

Reply:
thought about that too but after getting this merged, I was thinking of adding the form on other non-blog pages.  then each parent component will need to pass the `MAILCHIMP_URL` as a prop, which gets super messy.

thoughts on env var?  or just keeping this URL in this component?  imagine having this component in a bunch of different templates in the future.

Reply:
I'd just leave it hard-coded. It's not really a secret nor are we trying to make this component reusable for multiple lists atm. It's trivial to refactor later.